homeworks/rostyslav.havuka_RostyslavHavuka/5-rxjs/main.js

```diff
@@ -0,0 +1,20 @@
+const { fromEvent, animationFrameScheduler } = window.rxjs;
+const {
+    pairwise, map, distinctUntilChanged, throttleTime,
+} = window.rxjs.operators;
+
+const scroll = fromEvent(window, 'scroll');
+const header = document.querySelector('header');
```

Comment:
I believe you can have many headers on the page. It's better to search for your specific header by using `data-attributes`:

```html
<header class="logo" role="banner" data-header>
  <h1>Logo</h1>
</header>
```

```js
const header = document.querySelector('[data-header]');
```